Pusher.cpp

```diff
@@ -30,8 +30,9 @@
 #include <string.h>
 #include <unistd.h>
 
-Pusher::Pusher(string cerFileName) {
+Pusher::Pusher(string cerFileName, string passWord) {
```

Comment:
We should use the optional param here `string password = ""`

nit: password not passWord

Thank you very much for submitting the request!